controllers/seed-controller.js

```diff
@@ -1,75 +1,90 @@
-const {fetchTopics, fetchArticleById, updateArticle, fetchUsers, fetchArticles, fetchArticleComments, checkArticleExists, insertComment, checkUserExists} = require('../models/seed-models')
+const {
+  fetchTopics,
+  fetchArticleById,
+  updateArticle,
+  fetchUsers,
+  fetchArticles,
+  fetchArticleComments,
+  checkArticleExists,
+  insertComment,
+  checkUserExists,
+} = require("../models/seed-models");
 
 exports.getTopics = (req, res, next) => {
-    fetchTopics().then((topics) => {
-        res.status(200).send({topics})
+  fetchTopics()
+    .then((topics) => {
+      res.status(200).send({ topics });
     })
     .catch((err) => {
-        next(err)
-    })
-}
+      next(err);
+    });
+};
 
 exports.getArticleById = (req, res, next) => {
-    const {article_id} = req.params
-    fetchArticleById(article_id).then((article) => {
-        res.status(200).send({article})
+  const { article_id } = req.params;
+  fetchArticleById(article_id)
+    .then((article) => {
+      res.status(200).send({ article });
     })
     .catch((err) => {
-        next(err)
-    })
-}
+      next(err);
+    });
+};
 
-exports.patchArticle = (req, res) => {
-    const articleId = (req.params.article_id);
-    if (typeof req.body.votes === 'undefined'){
-        res.status(400).send({msg: "Missing required fields"});
-    } 
-    else if (typeof req.body.votes === 'number') {
-        updateArticle(articleId, req.body.votes).then((article) => {
-            res.status(200).send({article})
-        })
-    }
-    else {
-        res.status(400).send({msg: 'Bad request'})
-    }
-}
+exports.patchArticle = (req, res, next) => {
+  const articleId = req.params.article_id;
+  if (typeof req.body.votes === "undefined") {
+    res.status(400).send({ msg: "Missing required fields" });
```

Comment:
I would suggest Promise.rejecting in the model instead - simply because you can catch and pass to `next(err)` and make use of your error handlers rather than bloat the controller